src/root/rmem.d

```diff
@@ -127,7 +127,7 @@ else
     __gshared size_t heapleft = 0;
     __gshared void* heapp;
 
-    extern (C++) void* allocmemory(size_t m_size)
+    extern (C) void* allocmemory(size_t m_size)
```

Comment:
> I would prefer you changed the C++ side to use dsize_t instead.

Not another name debate, sigh.

> If you make it  extern(C) , it should have some sort of prefix to avoid name clashes, unless I'm missing something.

There aren't any other functions with that name, so no conflict.